sentry_sdk/utils.py

```diff
@@ -697,7 +697,8 @@ def single_exception_from_error_tuple(
     is_root_exception = exception_id == 0
     if not is_root_exception and parent_id is not None:
         exception_value["mechanism"]["parent_id"] = parent_id
-        exception_value["mechanism"]["type"] = "chained"
+        if exception_id is not None and exception_id > 0:
```

Comment:
[question] Given that we are already checking above that `is_root_exception` is `False`, which implies `exception_id != 0` (which, I believe is usually equivalent to `exception_id > 0` assuming it is not None since the ID should be nonnegative), can you explain the purpose of adding this conditional check?

Reply:
True. The if is probably useless.